app/src/test/java/org/mozilla/fenix/wallpapers/WallpapersUseCasesTest.kt

```diff
@@ -537,7 +535,6 @@ class WallpapersUseCasesTest {
         appStore.waitUntilIdle()
         assertEquals(selectedWallpaper.name, slot.captured)
         assertEquals(selectedWallpaper, appStore.state.wallpaperState.currentWallpaper)
-        assertEquals(selectedWallpaper.name, Wallpapers.wallpaperSelected.testGetValue()?.first()?.extra?.get("name")!!)
```

Comment:
This could be done as a folow-up, but I think removing these metrics tests means we can remove the glean test rule and also `@RunWith(AndroidJUnit4::class)`, which should significantly speed up the runtime for this class